apps/src/code-studio/pd/international_optin/InternationalOptin.jsx

```diff
@@ -0,0 +1,248 @@
+import React, {PropTypes} from 'react';
+import FormController from '../form_components/FormController';
+import FormComponent from '../form_components/FormComponent';
+import DatePicker from '../workshop_dashboard/components/date_picker';
+import moment from 'moment';
+import {DATE_FORMAT} from '../workshop_dashboard/workshopConstants';
+import {
+  Row,
+  Col,
+  ControlLabel,
+  FormGroup
+} from 'react-bootstrap';
+
+export default class InternationalOptin extends FormController {
+  static propTypes = {
+    accountEmail: PropTypes.string.isRequired,
+    labels: PropTypes.object.isRequired
+  };
+
+  /**
+   * @override
+   */
+  onSuccessfulSubmit(data) {
+    window.location = `/pd/international_optin/${data.id}/thanks`;
+  }
+
+  /**
+   * @override
+   */
+  serializeFormData() {
+    const formData = super.serializeFormData();
+    //Object.assign(formData['form_data'], this.getDistrictData());
+
+    return formData;
+  }
+
+  /**
+   * @override
+   */
+  getPageComponents() {
+    return [
+      InternationalOptinComponent
+    ];
+  }
+
+  /**
+   * @override
+   */
+  getPageProps() {
+    return {
+      ...super.getPageProps(),
+      accountEmail: this.props.accountEmail,
+      labels: this.props.labels
+    };
+  }
+}
+
+
+class InternationalOptinComponent extends FormComponent {
+  static propTypes = {
+    accountEmail: PropTypes.string.isRequired
```

Comment:
Are we going to pre-fill the name as well?

Reply:
Good idea.  Is teacher name always available as separate first & last name fields on `User`?  Is that what `ops_first_name` and `ops_last_name` are?

Reply:
No, `User.name` is whatever the user provides for Display Name in account settings. 

`ops_first_name` and `ops_last_name` were manually supplied by our internal "ops" team via the old / deprecated ops workshop dashboard. These are not reliable and not up to date.

Reply:
Good to know.  We won't pre-fill the name then, since we're not sure how to parse the Display Name afaik.